server/tests/ballot_comparison/snapshots/snap_test_ballot_comparison.py

```diff
@@ -41,8 +41,8 @@
 \r
 ######## ROUNDS ########\r
 Round Number,Contest Name,Targeted?,Sample Size,Risk Limit Met?,P-Value,Start Time,End Time,Audited Votes\r
-1,Contest 1,Targeted,11,No,1.1043717961,DATETIME,DATETIME,Choice 1-1: 4; Choice 1-2: 5\r
-1,Contest 2,Opportunistic,,No,0.4183843254,DATETIME,DATETIME,Choice 2-1: 4; Choice 2-2: 4; Choice 2-3: 0\r
+1,Contest 1,Targeted,11,No,0.7413124393,DATETIME,DATETIME,Choice 1-1: 4; Choice 1-2: 5\r
+1,Contest 2,Opportunistic,,No,0.3142300101,DATETIME,DATETIME,Choice 2-1: 4; Choice 2-2: 4; Choice 2-3: 0\r
```

Comment:
why did the p-values change here?

Reply:
Hmmmmm that's a really good question.

Reply:
Looks like I forgot to add testing of the p-value calculations. I will work on that and continue to chase this down.

Reply:
Okay, this issue should be resolved. There was a bug in the p-value calculations that is now fixed, so the snapshot will still change.